### PR TITLE
Make Github recognize licensing - v2

### PR DESCRIPTION
**Pull request by dals83**

### CODE REVIEW

File: COPYRIGHT [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-# License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful, but
-# WITHOUT ANY WARRANTY; without even the implied warranties of
-# MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
-# PURPOSE.  See the GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-~
```
Comment (magenta-af) [inline COPYRIGHT]: I realise it isn't exactly fair to ask you to answer questions about a commit you didn't write, but I don't understand why this is necessary. If we're going to remove this, might we perhaps want to replace it with a proper `AUTHORS` file?

**Review by ghost — COMMENTED** [review]
Basically uncontroversial! ... with one exception...

File: COPYRIGHT (hunk 1 above)
Comment (dals83) [reply COPYRIGHT]: That is a fair point you raise. I think the best solution would be to add it as an AUTHORS file instead.

Comment (magenta-af) [reply COPYRIGHT]: OK; in that case, let's merge this for now, and then I'll add that in another PR.

**Review by ghost — APPROVED** [review]

**Pull request merged**